components/tools/OmeroWeb/omeroweb/webgateway/views.py

```diff
@@ -2514,3 +2540,323 @@ def object_table_query(request, objtype, objid, conn=None, **kwargs):
     tableData['parentId'] = ann['parentId']
     tableData['addedOn'] = ann['addedOn']
     return tableData
+
+
+def build_url(request, name, api_version, **kwargs):
```

Comment:
Added docs and made ```build_url``` configurable in https://github.com/openmicroscopy/openmicroscopy/pull/4708/commits/446754fce3b40fdf9735d1d98aa486281c3aec99